src/App.js

```diff
@@ -30,27 +30,33 @@ class App extends Component {
     };
 
     // Uncomment this to spawn a single test item
-    // const testItem = this.spawnItem(Date.now());
-    // this.state.items.push(testItem);
+    const testItem = this.spawnItem(Date.now());
+    this.state.items.push(testItem);
 
     // Uncomment this to automatically spawn new items
     this.enableSpawner();
 
     console.log(this.state);
   }
 
-  onItemClicked = () => {
+  onItemClicked = (type) => {
```

Comment:
You don't need to pass in the type necessarily! You could instead use logic at the `GameItem` level to decide whether or not this function gets called.